src/preprocess.py

```diff
@@ -411,9 +411,23 @@ def _get_task(name, args, data_path, scratch_path):
             tokenizer_name=args.tokenizer,
             **task_kw,
         )
+        task.load_data()
         utils.maybe_make_dir(os.path.dirname(pkl_path))
         pkl.dump(task, open(pkl_path, "wb"))
-    # task.truncate(max_seq_len, SOS_TOK, EOS_TOK)
+
+    return task
+
+
+def get_skeletal_task(task_name, args):
+    """ Build a task without loading data """
```

Comment:
Explain when you would want to build a task without loading data

Reply:
Currently the intended use is for instantiating parts of models (based on a task) without needing to also load the data (this functionality doesn't exist yet, but will depend on this PR). I'm not sure if we should include that in the description, as it may ultimately also be used for other purposes.